modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -54,5 +61,50 @@
     $("[name=cleardef]").blur();
   });
   <{/if}>
+  
+  <{if $content.type eq "select"}>
+  var i=0;
+  var length=<{$content.useroptions|@count}>;
+  for(i=0;i<length;i++){
+    if (($("[name=ele_value["+i+"]]").val()=="{USERNAMES}" ||$("[name=ele_value["+i+"]]").val()=="{FULLNAMES}")&&$("[name=defaultoption["+i+"]]").is(":checked")){

```

Comment:
It is good that the option to allow adding new entries is disabled, but instead of this logic here, I would have made a change in the code that saves the element settings into the database. When saving the settings, if one of the names options is selected then automatically switch to not allowing users to add new options.